vapi/namespace/namespace.go

```diff
@@ -94,3 +171,35 @@ func (c *Manager) SupportBundleRequest(ctx context.Context, bundle *SupportBundl
 
 	return http.NewRequest(http.MethodPost, bundle.URL, &b)
 }
+
+type DistributedSwitchCompatibilitySummary struct {
+	Compatible        bool   `json:"compatible"`
+	DistributedSwitch string `json:"distributed_switch"`
+}
+
+func (c *Manager) ListCompatibleDistributedSwitches(ctx context.Context, clusterId string) (result []DistributedSwitchCompatibilitySummary, err error) {
+	listUrl := c.Resource(internal.NamespaceDistributedSwitchCompatibility).
+		WithParam("cluster", clusterId).
+		WithParam("compatible", "true")
+	if err != nil {
```

Comment:
Don't see how err could `!= nil` here?

Reply:
This was an oversight, I did a couple of iterations on the changes to WithParam. Now it swallows the error returned by ParseQuery, so these can be removed